druid/src/theme.rs

```diff
@@ -74,9 +74,8 @@ pub const SCROLLBAR_EDGE_WIDTH: Key<f64> =
     Key::new("org.linebender.druid.theme.scrollbar_edge_width");
 
 /// An initial theme.
-pub fn init() -> Env {
-    Env::default()
-        .adding(WINDOW_BACKGROUND_COLOR, Color::rgb8(0x29, 0x29, 0x29))
+pub fn init(env: Env) -> Env {
```

Comment:
I would do something slightly fancy here, since ultimately I think we want to make this private API:

- keep the function signature the same (`pub fn init() -> Env`)
- the impl should just call `Env::default()`
- add `#[deprecated(since = "0.7.0", note = "use Env::default() instead")]`
- add a new `pub (crate) fn` called something like `add_to_env` that is exactly what you have, here


Does that make sense?